src/status_im/chat/views/toolbar_content.cljs

```diff
@@ -67,7 +67,7 @@
                 contacts
                 chat-id
                 public?]}
-        (subscribe [:chat-properties [:group-chat :name :contacts :chat-id :public?]])
+        @(subscribe [:chat-properties [:group-chat :name :contacts :chat-id :public?]])
```

Comment:
For this first step I tried not to make changes but focus on migration to latest `re-frame`.